src/valueMappingsWrapper/index.ts

```diff
@@ -0,0 +1,20 @@
+import { getMappedValue } from './v7/valueMappings';
+import { getValueMappingResult } from './v8/valueMappings';
+
+//
+// To maintain compatibility back to Grafana >=v7.3, this wrapper is needed
+// to process the value mappings based on the object passed in.
+//
+// the method `getMappedValue()` function was removed in v9
+// the replacement `getValueMappingResult()` only appears in v8.1.0+
+//
+// At runtime, this cannot be determined so the legacy function and the new function are both provided
+// here, to allow this to work work across all versions (7.3 > 9.x)
+//
+export const GetMappedValue = (valueMappings: any[], value: any): any => {
+  let mappedValue = getMappedValue(valueMappings, value);
+  if (typeof mappedValue === 'undefined') {
+    return getValueMappingResult(valueMappings, value);
+  }
+  return mappedValue;
+};
```

Comment:
Nit: maybe you can use the [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) as well?

```suggestion
return getMappedValue(valueMappings, value) ?? getValueMappingResult(valueMappings, value);
};
```